src/games/strategy/triplea/ui/UIContext.java

```diff
@@ -32,7 +33,7 @@
 /**
  * A place to find images and map data for a ui.
  */
-public class UIContext extends AbstractUIContext implements IUIContext {
```

Comment:
In this case AbstractUIContext or one of its superclasses is already implementing IUIContext